src/components/NcBreadcrumbs/NcBreadcrumbs.vue

```diff
@@ -547,7 +547,6 @@ export default {
 				ref: 'actionsBreadcrumb',
 				key: 'actions-breadcrumb-1',
 				// Add handlers so the Actions menu opens on hover
-				onDragstart: this.dragStart,
```

Comment:
Why should this be removed? Doesn't it act the same as in Vue 2 here?

Reply:
> Why should this be removed? Doesn't it act the same as in Vue 2 here?

It is redundant, and could be removed for `master` as well.

`dragStart` in `NcBreadcrumbs`:
https://github.com/nextcloud-libraries/nextcloud-vue/blob/d8d2621ca341e3d7e02bab8ec77c36ea3bbd854f/src/components/NcBreadcrumbs/NcBreadcrumbs.vue#L363-L365

and we set it anyway to
https://github.com/nextcloud-libraries/nextcloud-vue/blob/d8d2621ca341e3d7e02bab8ec77c36ea3bbd854f/src/components/NcBreadcrumb/NcBreadcrumb.vue#L37
in `NcBreadcrumb`.